tutorials/hoon/bank-account.md

```diff
@@ -75,6 +75,6 @@ Each of these arms produces a gate which takes an `@ud` argument. Each of these
 +>.$(balance (add balance amount))
 ```
 
-`+>` is [wing syntax](/docs/reference/hoon-expressions/limb/limb/). This particularly wing construction looks for the the tail of the tail (the third element) in `$`, the subject of the gate we are in, which is the entire `new-account` door. We change `balance` to be the result of adding `balance` and `amount` and produce the door as the result. `withdraw` functions the same way only doing subtraction instead of addition.
+`+>` is [wing syntax](@/docs/reference/hoon-expressions/limb/wing/). This particularly wing construction looks for the the tail of the tail (the third element) in `$`, the subject of the gate we are in, which is the entire `new-account` door. We change `balance` to be the result of adding `balance` and `amount` and produce the door as the result. `withdraw` functions the same way only doing subtraction instead of addition.
```

Comment:
Is this the right link syntax?

Reply:
No, needs to end with `.md`. You're linking to the file's relative path, and Zola converts it to the permalink.

Reply:
I thought so. @syzygyzer, can you fix this please?